source/index.html.md.erb

```diff
@@ -21,5 +21,6 @@ dashboard](https://www.gov.uk/performance/govuk-pay).
 
 ## How much it costs
 
-There is no charge to use GOV.UK Pay. You will still need to pay your payment
-service provider’s (PSP’s) transaction fees.
+There is no charge to use GOV.UK Pay.
+
+You will still need to pay your payment service provider’s (PSP’s) transaction fees. If you use GOV.UK Pay's PSP, GOV.UK Pay will automatically deduct the transaction fee from each user payment.
```

Comment:
is this meant to say user's payment? i wonder if the inclusion of 'user' is potentially a bit confusing since it could suggest it affects what the user thinks they're paying for?

Reply:
Thanks Katie - I've removed "user".